deployment/modules/gcp/cloudbuild/preloaded/main.tf

```diff
@@ -0,0 +1,136 @@
+  conformance_gcp_docker_image = "${local.artifact_repo}/conformance-gcp"
+}
+
+resource "google_project_service" "cloudbuild_api" {
+  service            = "cloudbuild.googleapis.com"
+  disable_on_destroy = false
+}
+
+## Service usage API is required on the project to enable APIs.
+## https://cloud.google.com/apis/docs/getting-started#enabling_apis
+## serviceusage.googleapis.com acts as a central point for managing the API 
+## lifecycle within your project. By ensuring the required APIs are enabled 
+## and accessible, it allows Cloud Build to function seamlessly and interact 
+## with other Google Cloud services as needed.
+## 
+## The Cloud Build service account also needs roles/serviceusage.serviceUsageViewer.
+resource "google_project_service" "serviceusage_api" {
+  service            = "serviceusage.googleapis.com"
+  disable_on_destroy = false
+}
+
+resource "google_cloudbuild_trigger" "build_trigger" {
+  name            = "build-docker-${var.docker_env}"
+  service_account = "projects/${var.project_id}/serviceAccounts/${local.cloudbuild_service_account}"
+  location        = var.location
+
+  # TODO(phboneff): use a better mechanism to trigger releases that re-uses Docker containters, or based on branches rather.
+  # This is a temporary mechanism to speed up development.
+  github {
+    owner = var.github_owner
+    name  = "static-ct"
+    push {
+      tag = "^staging-deploy-(.+)$"
+    }
+  }
+
+  build {
+    ## Build the SCTFE GCP Docker image.
+    ## This will be used by the building the conformance Docker image which includes 
+    ## the test data.
+    step {
+      id   = "docker_build_sctfe_gcp"
+      name = "gcr.io/cloud-builders/docker"
+      args = [
+        "build",
+        "-t", "sctfe-gcp:$SHORT_SHA",
+        "-t", "sctfe-gcp:latest",
+        "-f", "./cmd/gcp/Dockerfile",
+        "."
+      ]
+    }
+
+    ## Build the SCTFE GCP Conformance Docker container image.
+    step {
+      id   = "docker_build_conformance_gcp"
+      name = "gcr.io/cloud-builders/docker"
+      args = [
+        "build",
+        "-t", "${local.conformance_gcp_docker_image}:$SHORT_SHA",
+        "-t", "${local.conformance_gcp_docker_image}:latest",
+        "-f", "./cmd/gcp/staging/Dockerfile",
+        "."
+      ]
+      wait_for = ["docker_build_sctfe_gcp"]
+    }
+
+    ## Push the conformance Docker container image to Artifact Registry.
+    step {
+      id   = "docker_push_conformance_gcp"
+      name = "gcr.io/cloud-builders/docker"
+      args = [
+        "push",
+        "--all-tags",
+        local.conformance_gcp_docker_image
+      ]
+      wait_for = ["docker_build_conformance_gcp"]
+    }
+
+    ## Apply the deployment/live/gcp/static-staging/logs/arche2025h1 terragrunt config.
+    ## This will bring up the conformance infrastructure, including a service
+    ## running the conformance server docker image built above.
```

Comment:
```suggestion
    ## This will bring up or update the conformance infrastructure, including a service
    ## running the conformance server docker image built above.
```